eng/pipelines/coreclr/exploratory.yml

```diff
@@ -0,0 +1,63 @@
+# This job definition automates the exploratory tool.
+
+trigger:
+  batch: false
```

Comment:
Do we really want this triggered with every JIT change? That might be useful, but potentially is overkill?

Reply:
Sounds reasonable. We might not even have to triage the incoming issues. Changed it to trigger nightly and will readjust depending on how useful this proves.